.github/ci-test-connections.yaml

```diff
@@ -139,3 +139,11 @@ connections:
       aws_access_key_id: "ROOTNAME"
       aws_secret_access_key: "CHANGEME123"
       endpoint_url: "http://127.0.0.1:9000"
+  - conn_id: mysql_conn
+    conn_type: mysql
+    host: $MYSQL_HOST
+    login: $MYSQL_LOGIN
+    password: $MYSQL_PASSWORD
+    port: 3306
+    schema: $MYSQL_DB
+    extra:
```

Comment:
We can remove `extra` if we don't have values for it

Reply:
done